jupyterhub/orm.py

```diff
@@ -163,28 +163,32 @@ def __repr__(self):
 # lots of things have roles
 # mapping tables are the same for all of them
 
-_role_map_tables = []
+role_associations = {}
```

Comment:
this should remain private

Reply:
Done in 5aa8d299134902ee2f19f5d40cf822589cc5d44a.